dlib/image_loader/jxl_loader.h

```diff
@@ -23,6 +23,14 @@ namespace dlib
         jxl_loader(const dlib::file& f);
         jxl_loader(const unsigned char* imgbuffer, size_t buffersize);
 
+        bool is_gray() const;
+        bool is_graya() const;
+        bool is_rgb() const;
+        bool is_rgba() const;
+        unsigned int bit_depth() const;
+        unsigned int nr() const;
+        unsigned int nc() const;
```

Comment:
```suggestion
        long nr() const;
        long nc() const;
```
Any objection to this?  That way it's consistent with all the other `nr()` and `nc()` functions.  Mostly doesn't matter, but someone will eventually do `nr()-something` and get surprised it's unsigned at some point.  Or they will get compiler warnings about signed vs unsigned comparisons and be annoyed by it.

Reply:
Actually, I implemented it like this at the beginning, but somehow I saw all the other methods returned unsigned int, so I changed it. But yeah, long is better.